src/components/atoms/bar-chart-content.tsx

```diff
@@ -120,41 +144,88 @@ export const BarChartContent: FC<BarChartContentProps> = ({
-      spacingOuter={gapPercent / 100}
-      contentInset={{
-        top: contentInset.top + cornerRoundness,
-        bottom: contentInset.bottom - cornerRoundness
-      }}
-      yMax={yMax}
-      svg={{
-        fill: secondaryColor
-      }}>
-      <Grid
-        svg={{
-          y: 0 - cornerRoundness,
-          strokeWidth: 1,
-          stroke: colors.dot,
-          strokeDasharray: [5, 3],
-          strokeDashoffset: 0
+    <View accessible>
+      <BarChart
+        style={style}
+        data={chartData.map((value: number, index: number) =>
+          index === chartData.length - 1
+            ? {value, svg: {fill: colors.purple}}
+            : {value}
+        )}
+        yAccessor={({item}: {item: {value: number}}) => item.value}
+        gridMin={0}
+        scale={scale}
+        numberOfTicks={3}
+        spacingInner={gapPercent / 100}
+        spacingOuter={gapPercent / 100}
+        contentInset={{
+          top: contentInset.top + cornerRoundness,
+          bottom: contentInset.bottom - cornerRoundness
         }}
-      />
-      {/* @ts-ignore: gets BarChildProps from BarChart parent */}
-      <RoundedBarToppers />
-      {/* @ts-ignore: gets BarChildProps from BarChart parent */}
-      <XAxisTrim />
-      {/* @ts-ignore: gets BarChildProps from BarChart parent */}
-      {showTrendLine && <BackgroundTrendLine />}
-      {/* @ts-ignore: gets BarChildProps from BarChart parent */}
-      {showTrendLine && <ForegroundTrendLine />}
-    </BarChart>
+        yMax={yMax}
+        svg={{
+          fill: secondaryColor
+        }}>
+        {<Label />}
+        <Grid
+          svg={{
+            y: 0 - cornerRoundness,
+            strokeWidth: 1,
+            stroke: colors.dot,
+            strokeDasharray: [5, 3],
+            strokeDashoffset: 0
+          }}
+        />
+        {/* @ts-ignore: gets BarChildProps from BarChart parent */}
+        {<RoundedBarToppers />}
+        {/* @ts-ignore: gets BarChildProps from BarChart parent */}
+        {<XAxisTrim />}
+        {/* @ts-ignore: gets BarChildProps from BarChart parent */}
+        {showTrendLine && <BackgroundTrendLine />}
+        {/* @ts-ignore: gets BarChildProps from BarChart parent */}
+        {showTrendLine && <ForegroundTrendLine />}
+      </BarChart>
+    </View>
   );
 };
+
+const styles = StyleSheet.create({
+  label: {
+    borderStyle: 'solid',
+    borderWidth: 1,
+    borderColor: '#ACAFC4',
+    position: 'absolute',
+    right: -10,
+    minWidth: 60,
+    borderRadius: 5,
+    backgroundColor: 'white',
+    zIndex: 30
+  },
+  triangle: {
+    width: 5,
+    height: 10,
+    position: 'absolute',
+    bottom: -10,
+    left: 36,
+    borderLeftWidth: 7,
+    borderLeftColor: 'transparent',
+    borderRightWidth: 7,
+    borderRightColor: 'transparent',
+    borderTopWidth: 7,
+    borderTopColor: '#ACAFC4'
+  },
+  triangle2: {
+    width: 5,
+    height: 10,
+    position: 'absolute',
+    bottom: -10,
```

Comment:
`-9.75` here covers the hairline on my device and makes the apparent line width more even